pkg/virtual/workspaces/authorization/watch_test.go

```diff
@@ -62,7 +62,7 @@ func newTestWatcher(username string, groups []string, predicate storage.Selectio
 	informers := tenancyInformers.NewSharedInformerFactory(mockClient, controller.NoResyncPeriodFunc())
 	workspaceCache := workspacecache.NewClusterWorkspaceCache(
 		informers.Tenancy().V1alpha1().ClusterWorkspaces().Informer(),
-		&mockClusterClient{mockClient: mockClient},
```

Comment:
is mockClusterClient now unused? Can we remove it?

Reply:
Yes, it was just wrapping the `Kubermetes.Interface` with `.Cluster()`. We don't need it anymore